buildlib/coverity.yml

```diff
@@ -0,0 +1,38 @@
+parameters:
+  demands: []
+  container: rhel76
+  modes: ["release", "devel"]
+
+jobs:
+  - ${{each mode in parameters.modes }}:
+    - job: coverity_${{ mode }}
+      pool:
+        name: MLNX
+        demands: ${{ parameters.demands }}
+      displayName: coverity ${{ mode }} on ${{ parameters.container }}
+      container: ${{ parameters.container }}
+      timeoutInMinutes: 360
+      steps:
+        - checkout: self
+          clean: true
+        - bash: |
+            . /etc/profile.d/modules.sh
+            export MODULEPATH="/hpc/local/etc/modulefiles:$MODULEPATH"
+            source ./buildlib/az-helpers.sh
+            ./contrib/tools/coverity.sh ${{ mode }}
+            res=$?
+            reportExists=False
+            test -f $(System.DefaultWorkingDirectory)/cov_build_${{ mode }}_1/output/errors/index.html && reportExists=True
+            echo "##vso[task.setvariable variable=reportExists]$reportExists"
+            if [[ $res -eq 0 ]] ; then
+              echo "##vso[task.complete result=Succeeded;]DONE"
+            else
+              echo "##vso[task.complete result=Failed;]Coverity have errors"
+            fi
+
+          displayName: ${{ mode }}
+        - task: PublishPipelineArtifact@1
+          inputs:
+            targetPath: $(System.DefaultWorkingDirectory)/cov_build_${{ mode }}_1/output/errors/index.html
```

Comment:
1. maybe add a variable for `$(System.DefaultWorkingDirectory)/cov_build_${{ mode }}_1/output/errors`
2. IMO, need to pack whole directory, not just index.html